webpack.config.js

```diff
@@ -18,26 +24,62 @@ module.exports = {
+      Path.join(__dirname, "node_modules/react-intl/dist/react-intl.min.js"),
+      Path.join(__dirname, "node_modules/react-router/umd/ReactRouter.min.js"),
+      Path.join(__dirname, "node_modules/react-ga/dist/index.min.js"),
+      Path.join(__dirname, "pages/sequential.jsx")
+    ],
     loaders: [
-      { test: /\.js$/, loaders:  ['babel-loader'], exclude: ['node_modules'] },
-      { test: /\.jsx$/, loaders: ['babel-loader'], exclude: ['node_modules'] },
-      { test: /\.json$/, loaders: ['json-loader'], exclude: ['node_modules'] },
+      { test: /\.js$/, loaders:  ['babel-loader'], exclude: exclude },
+      { test: /\.jsx$/, loaders: ['babel-loader'], exclude: exclude },
+      { test: /\.json$/, loaders: ['json-loader'], exclude: exclude },
       { test: /\.less$/, loader: ExtractTextPlugin.extract(
                     'css?sourceMap!less?sourceMap'
-                ), exclude: ['node_modules'] }
+                ), exclude: exclude }
     ],
     preLoaders: [
-      { test: /\.jsx$/, loaders: ['eslint-loader'], exclude: ['node_modules'] }
+      { test: /\.jsx$/, loaders: ['eslint-loader'], exclude: exclude }
     ]
   },
+  resolve: {
+    alias: {
+      "react$": Path.join(__dirname, "node_modules/react/dist/react.min.js"),
+      "react-intl$": Path.join(__dirname, "node_modules/react-intl/dist/react-intl.min.js"),
+      "react-router$": Path.join(__dirname, "node_modules/react-router/umd/ReactRouter.min.js"),
+      "react-ga$": Path.join(__dirname, "node_modules/react-ga/dist/index.min.js")
+    }
+  },
   eslint: {
     emitError: true,
     emitWarning: true
   },
+  devtool: 'inline-eval-cheap-source-map',
   plugins: [
     new AssetsPlugin({
       path: Path.join(__dirname, "public")
     }),
+    new webpack.PrefetchPlugin("./mixins/form.jsx"),
+    new webpack.PrefetchPlugin("./assets/js/analytics.js"),
+    new webpack.PrefetchPlugin("./data/pages.js"),
+    new webpack.PrefetchPlugin("./less/index.less"),
+    new webpack.PrefetchPlugin("./pages/one-page.jsx"),
+    new webpack.PrefetchPlugin("./pages/sequential.jsx"),
+    new webpack.PrefetchPlugin("./pages/paypal-donate.jsx"),
+    new webpack.PrefetchPlugin("./public/locales.json"),
+    new webpack.PrefetchPlugin("./components/client.jsx"),
+    new webpack.PrefetchPlugin("./scripts/langURLParser.js"),
+    new webpack.PrefetchPlugin("./scripts/queryParser.js"),
+    new webpack.PrefetchPlugin("./locales/i18n.js"),
+    new webpack.PrefetchPlugin("./pages/share.jsx"),
+    new webpack.PrefetchPlugin("./pages/thunderbird/one-page.jsx"),
+    new webpack.PrefetchPlugin("./pages/thank-you.jsx"),
+    new webpack.PrefetchPlugin("./pages/thunderbird/thank-you.jsx"),
+    new webpack.PrefetchPlugin("./components/footer-mozilla.jsx"),
+    new webpack.PrefetchPlugin("./components/small-print.jsx"),
+    new webpack.PrefetchPlugin("./components/name-input.jsx"),
+    new webpack.PrefetchPlugin("./components/donation-frequency.jsx"),
```

Comment:
I couldn't see a different with or without these, the the webpack profile was pretty adamant that it would help.